network-api/networkapi/buyersguide/factory.py

```diff
@@ -33,6 +34,8 @@ class Meta:
 
     draft = Faker('boolean')
     adult_content = Faker('boolean')
+    review_date = Faker('date_time_between_dates',
+                        datetime_start=date(year=2018, month=11, day=1), datetime_end=None, tzinfo=timezone.utc)
```

Comment:
Note this is why all the products on review app have random `review_date` values instead of November 1, 2018. Let me know if should update factory as a follow-up PR instead.